src/main/java/com/resturant/menu/models/Item.java

```diff
@@ -19,22 +22,49 @@
     String internalDescription;
     Double price;
     String updated;
-    
+
+    HashMap img;
+
+    public HashMap getImg() {
+        return img;
+    }
+
+    public void setImg(HashMap img) {
+        this.img = img;
+    }
+
+    @Override
+    public String toString() {
+        return "Item{" +
+                "id='" + id + '\'' +
+                ", name='" + name + '\'' +
+                ", description='" + description + '\'' +
+                ", sectionId='" + sectionId + '\'' +
+                ", internalDescription='" + internalDescription + '\'' +
+                ", price=" + price +
+                ", updated='" + updated + '\'' +
+                ", img=" + img +
+                ", image=" + image +
+                ", options=" + Arrays.toString(options) +
+                '}';
+    }
+
     @DBRef
     Image image;
```

Comment:
if we are changing the implementation and not adding images as a reusable resource (like items or sections) we should clean up this and remove the image controller / repos / ect

Reply:
or we could add an Images section and add dropdowns to select them for the other sections

Reply:
I can go thru and clean it up.

Reply:
Finished removing references.